CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
   and new formats, so old `jj` versions should be able to read these index
   files. This compatibility layer will be removed in a future release.
 
+* `jj op undo` is deprecated in favor of `jj op revert`.
```

Comment:
Just this morning, I wanted to undo a commit I had made in Breezy, which has `uncommit`. It leaves the working copy as is, so you can commit again. What I wanted was to get my file back to the previous commit. In Breezy, that is `revert`, which I think is `jj restore`.
I like `undo` since other VCSs don't use it.

Reply:
I'm not sure what your point is. But looking at that changelog entry again, I forgot to mention that `jj undo` is still available. I added it now.

Reply:
It's my two cents on the naming... Everyone knows what `undo` means, whereas `revert` seems to be different in every VCS.

Reply:
I see. We already have a command `jj revert`, which applies the inverse of a commit. I think `jj op revert` applying the inverse of an operation is consistent UI.

Reply:
@senekor because `jj undo` creates a new op entry with the contents of the second-to-last op revision, its behavior is identical to what you're describing. So `jj op undo` has the same "apply the reverse of the last commit" behavior that `git revert` or (apparently) breezy `revert` has.

Reply:
> So `jj op undo` has the same "apply the reverse of the last commit" behavior that `git revert` or (apparently) breezy `revert` has.

To be clear, Breezy `revert` does not affect the stored history. It only affects the working copy. I don't know what Git `revert` does, or whether that is applicable in Piper land (or other future backends).

Reply:
Andrew explained that `jj revert` behaves exactly the same way `git revert` does. And `jj op revert` behaves also the same way - just for operations.

Reply:
> Andrew explained that `jj revert` behaves exactly the same way `git revert` does.

You can't count explanations in GitHub comments. The user doesn't see those. And you also can't explain in terms of other VCSs, which not all users are familiar with.
> trying to match other VCS's

To be clear, I don't want to match them, especially when they use the same word differently.
I think we might all have different concepts of what software does, as we are all exposed to a different set of it. Those of us that are older might have ingrained notions from early software, and those in different fields are exposed to other jargon meaning different things.
This particular edit is for the CHANGELOG, so it should make sense for historical purposes why this whole change is made.

Reply:
Are you suggesting an edit to the changelog? You didn't say that anywhere until now. It seemed like you were just not in agreement with the choice of name.

Reply:
I chose this particular CHANGELOG edit to make my original comment because it was relevant to that. The entry itself is pretty innocuous, not giving any context of this change to the big picture or pointing to where the decision was made or anything (I don't know if those are wanted or required, just sayin'.)

Reply:
Since we already have `jj revert`, I think it makes sense to call this `jj op revert`. We can decide whether we should rename them both (to e.g. `jj invert` and `jj op invert`) but I don't think we should name them inconsistently now, and I don't want to block this PR on the naming discussion.

(It would be a bit unfortunate to rename `jj revert` so soon after we renamed it to that name from `jj backout` (in #5932)).

Reply:
It's starting to sound like adding more commands to do the same thing. So it's `revert`, `undo`, and `op revert` all doing the same thing? Is this temporary while deprecation time passes, or what? (this should be in the LOG entry)

Reply:
`revert` creates a commit that does the inverse of another commit. `op revert` creates an operation that does the inverse of another operation. `undo` currently does the same thing as `op revert` but we plan on making it different (as the suggested changelog snippet says).